app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreenViewModel.kt

```diff
@@ -0,0 +1,46 @@
+package com.wire.android.ui.userprofile.other
+
+import android.util.Log
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.lifecycle.SavedStateHandle
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.wire.android.navigation.NavigationManager
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+
+@HiltViewModel
+class OtherUserProfileScreenViewModel @Inject constructor(
+    savedStateHandle: SavedStateHandle,
+    private val navigationManager: NavigationManager
+) : ViewModel() {
+
+    private companion object {
+        const val USER_ID = "user_id"
+    }
+
+    var state: OtherUserProfileState by mutableStateOf(OtherUserProfileState())
+
+    init {
+        savedStateHandle.get<String>(USER_ID)?.let { id ->
+            Log.d("TEST", "user id $id")
```

Comment:
```suggestion
           appLogger.d("user id $id")
```
I think we can use our platform logger `appLogger`